constants/__init__.py

```diff
@@ -335,15 +335,13 @@
 # validator scoring exponential temperature
 # 0.01 gives ~96% to best model with only ~3 receiving any weights.
```

Comment:
Should we consider a different temperature for multi choice? How does it look if the deviations are like 0.55 vs 0.52 vs 0.5?

Reply:
As discussed, this is going to be based on how the deviations compare to the other models overall. So in a two model situation would be 1 and 0. With more the spread should still be reasonable enough to respond well to the current temperature.